res/values-es/strings.xml

```diff
@@ -726,9 +726,9 @@
     <string name="Refunded">"Reembolsada"</string>
 
 
-    <string name="contribution_warning">"¿Estás seguro de que quieres donar este plasma? Las donaciones no serán devueltas."</string>
+    <string name="contribution_warning">"¿Estás seguro de que quieres donar este plasma?, Las donaciones no serán devueltas."</string>
     <string name="Upload">"Subir"</string>
-    <string name="upload_warning">"¿Estás seguro de que deseas subir esta skin? Otros jugadores serán capaces de verla. Puede demorarse hasta 1 semana en ser revisada. Las skins inapropiadas serán rechazadas y el plasma no será devuelto."</string>
+    <string name="upload_warning">"¿Estás seguro de que deseas subir esta skin?, Otros jugadores serán capaces de verla. Puede demorarse hasta 1 semana en ser revisada. Las skins inapropiadas serán rechazadas y el plasma no será devuelto."</string>
```

Comment:
Lines 729 and 731: Please remove the commas because interrogation preceded by a comma is totally wrong.

Reply:
Cierto, error mío, cambios hechos